### DISCUSSION TITLE
calc 2.14.0.8 release

### DISCUSSION THREAD

**Discussion by lcn2** [lcn2/calc#38] [Announcements]
The following are the changes in this release:

    Identical to v2.14.0.7: now declared a stable release.

<hr /><em>This discussion was created from the release <a href='https://github.com/lcn2/calc/releases/tag/v2.14.0.8'>calc 2.14.0.8 release</a>.</em>

**Comment by pmetzger** [comment]
I'll update MacPorts ASAP.
